test/HighlightTest.php

```diff
@@ -39,4 +40,70 @@ public function testUnknownLanguageThrowsDomainException()
         $hl = new Highlighter();
         $hl->highlight("blah++", "als blurp eq z dan zeg 'flipper'");
     }
+
+    public function testListLanguagesWithoutAliases()
+    {
+        $expectedLanguageCount = count(glob(__DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR .
+                                            'Highlight' . DIRECTORY_SEPARATOR . 'languages' . DIRECTORY_SEPARATOR .
+                                            '*.json'));
+
+        $hl = new Highlighter();
+
+        $availableLanguages = $hl->listLanguages();
+        $this->assertEquals($expectedLanguageCount, count($availableLanguages));
+
+        $availableLanguages = $hl->listLanguages(false);
+        $this->assertEquals($expectedLanguageCount, count($availableLanguages));
+    }
+
+    public function testListLanguagesWithAliases()
+    {
+        $minimumLanguageCount = count(glob(__DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR .
+                                           'Highlight' . DIRECTORY_SEPARATOR . 'languages' . DIRECTORY_SEPARATOR .
+                                           '*.json'));
+
+        $hl = new Highlighter();
+        $availableLanguages = $hl->listLanguages(true);
+        $this->assertGreaterThan($minimumLanguageCount, count($availableLanguages));
```

Comment:
In addition to checking the count is different, could you add an assert to make sure a common alias does exist? For example:
- `c++` for `cpp`
- `yml` for `yaml`